src/betterproto/__init__.py

```diff
@@ -495,7 +590,35 @@ class ParsedField:
     raw: bytes
 
 
-def parse_fields(value: bytes) -> Generator[ParsedField, None, None]:
+def load_fields(stream: BinaryIO) -> Iterator[ParsedField]:
```

Comment:
```suggestion
def load_fields(stream: BinaryIO) -> Generator[ParsedField, None, None]:
```

Reply:
Reverted - this was just due to misreading the docs around this, I believed them to be equivalent here, but they are apparently not

Reply:
They will do basically the same thing to the end user but it's just more annoying if you ever have to go through the blame.